src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,11 +4,17 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  searchWord: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
+export const MoviesList: React.FC<Props> = ({ movies, searchWord }) => (
   <div className="movies">
-    {movies.map(movie => (
+    {movies.filter((movie) => {
+      return (movie.title.toLowerCase()
+        .includes(searchWord.toLowerCase())
+      || movie.description.toLowerCase()
+        .includes(searchWord.toLowerCase()));
+    }).map(movie => (
```

Comment:
This function belongs to parent component. Also create create constants f.e. `descriptionNormalized` and use it. Additional lines of code but a huge improvement in readability.